### ISSUE TITLE
'MinHashLSH' object has no attribute 'delete_session'

### ISSUE THREAD

**Issue by Priyabrata409** [ekzhu/datasketch#173]
This method already exists in Asynchronus LSH but not in MinHashLSH. MinHashLSH only has insertion session method

**Comment by ekzhu** [comment]
There is no need to delete if you use `with`. See [example](http://ekzhu.com/datasketch/lsh.html#minhash-lsh-at-scale)

**Comment by Priyabrata409** [comment]
I want to remove multiple keys in  batch, the way we are inserting using insertion_session. There is a delete_session method available on AsyncMinHashLSH but not in MinHashLSH. `async def func():
    async with AsyncMinHashLSH(storage_config=_storage, threshold=0.5, num_perm=16) as lsh:
        async with lsh.insertion_session(batch_size=1000) as session:
            fs = (session.insert(key, minhash, check_duplication=False) for key, minhash in data)
        await asyncio.gather(*fs)

        async with lsh.delete_session(batch_size=3) as session:
            fs = (session.remove(key) for key in keys_to_remove)
            await asyncio.gather(*fs)`

**Comment by ekzhu** [comment]
> I want to remove multiple keys in batch, the way we are inserting using insertion_session. There is a delete_session method available on AsyncMinHashLSH but not in MinHashLSH. ```
> 
> ```
> 
>         async with lsh.delete_session(batch_size=3) as session:
>             fs = (session.remove(key) for key in keys_to_remove)
>             await asyncio.gather(*fs)` 
> ```

Oh. Sorry I misunderstood the question. 

Yes currently there isn't a delete_session for MinHash LSH. This feature needs to be implemented for every type of storage. Would you be interested in taking a stab at this? Currently we support Cassandra and Redis as external storage. For in-memory MinHash LSH, I guess a session doesn't make it much faster.

**Comment by Priyabrata409** [comment]
Yes. It only requires for database integrated lsh. Sure i will raise a PR for this functionality
